src/app/bav/controllers/nameEntry.test.js

```diff
@@ -0,0 +1,12 @@
+const BaseController = require("hmpo-form-wizard").Controller;
+const { expect } = require("chai");
```

Comment:
I believe on the ticket we agreed to use jest instead of chai/mocha

Reply:
Jest tests have been added